addons/survey/views/survey_templates.xml

```diff
@@ -537,7 +577,7 @@
                 <div t-if="survey.scoring_type in ['scoring_with_answers', 'scoring_without_answers']">
                     <h1 class="mt16">Results Overview</h1>
                     <div>Success rate: <mark class="font-weight-bold"><t t-esc="survey_dict['success_rate']"></t>%</mark></div>
-                    <div id="scoring_results_chart" t-att-data-graph_data="survey_dict['scoring_graph_data']">
+                    <div class="survey_graph" data-graph_type="pie" t-att-data-graph_data="survey_dict['scoring_graph_data']">
```

Comment:
data-graph_type="pie"
Could be `data-graph-type="pie"` so you can do `$elem.data('graphType')`

(Ok, maybe I wrote that, but still!)

Reply:
I also took the opportunity to do `t-att-data-graph-data` instead of `t-att-data-graph_data`
#tankaffaire